src/pages/signup-affiliates/components/_personal-details.tsx

```diff
@@ -314,10 +314,10 @@ const PersonalDetails = ({
         })
     }, [personal_details])
 
-    const is_individual_validate =
+    const is_valid_individual =
```

Comment:
Can we conver these to truthy values and use the opposite in the rest of the code

Reply:
okay